src/AccessibilityInsights.Rules/Conditions/ValueCondition.cs

```diff
@@ -8,7 +8,7 @@
 namespace Axe.Windows.Rules
 {
     /// <summary>
-    /// This class enables comparisons such as ==, !=, >, and < in conditions.
+    /// This class enables comparisons such as ==, !=, &gt;, and &lt; in conditions.
```

Comment:
>&gt; [](start = 55, length = 4)

&gt; and &lt; will look right in intellisense but look odd here. Is there a way to make it work in both cases?

Reply:
I think we've finally found a knit too small to pick. :)

Reply:
"nit", not "knit"  :)

---
In reply to: [276857109](https://github.com/Microsoft/accessibility-insights-windows/pull/320#discussion_r276857109) [](ancestors = 276857109)